files/rsyslog/65-viaq-formatting.conf

```diff
@@ -96,6 +96,14 @@ if strlen($!_MACHINE_ID) > 0 then {
         set $!systemd!t!AUDIT_SESSION = $!_AUDIT_SESSION;
     }
     unset $!_AUDIT_SESSION;
+    if strlen($!_STREAM_ID) > 0 then {
+        set $!systemd!t!_STREAM_ID = $!_STREAM_ID;
+    }
+    unset $!_STREAM_ID;
+    if strlen($!_SYSTEMD_INVOCATION_ID) > 0 then {
+        set $!systemd!t!_SYSTEMD_INVOCATION_ID = $!_SYSTEMD_INVOCATION_ID;
+    }
+    unset $!_SYSTEMD_INVOCATION_ID;
```

Comment:
Please also add support for the new `systemd.t.LINE_BREAK` field - https://github.com/ViaQ/elasticsearch-templates/commit/aae0fcb56b282b2c8f2d49f7c1893dacd5a20d6f